WPFUI/Views/Tabs/Villages/MarketPage.xaml.cs

```diff
@@ -0,0 +1,46 @@
+using ReactiveUI;
+using Splat;
+using System.Reactive.Disposables;
+using WPFUI.ViewModels.Tabs.Villages;
+
+namespace WPFUI.Views.Tabs.Villages
+{
+    /// <summary>
+    /// Interaction logic for MarketPage.xaml
+    /// </summary>
+    public partial class MarketPage : ReactivePage<MarketViewModel>
+    {
+        public MarketPage()
+        {
+            ViewModel = Locator.Current.GetService<MarketViewModel>();
+            InitializeComponent();
+
+            SendOutLimit.ViewModel = new("Upper Limit:");
```

Comment:
remove " : ", it duplicate 
![image](https://user-images.githubusercontent.com/29301246/209904637-73b404f7-14ca-4ad5-b655-7799bef60e7a.png)